File: Razvoj mobilnih aplikacija labosi/Biography/app/src/main/java/com/filip/biography/MainActivity.java
package com.filip.biography;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.support.v7.widget.LinearLayoutManager;
import android.support.v7.widget.RecyclerView;

public class MainActivity extends AppCompatActivity {
    private RecyclerView recyclerView;
    private RecyclerView.LayoutManager layoutManager;

    private int[] images = {R.drawable.download, R.drawable.swh1web};

    private RecyclerAdapter recyclerAdapter;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        recyclerView = (RecyclerView) findViewById(R.id.recyclerviewID);
        layoutManager = new LinearLayoutManager(this);
        recyclerView.setLayoutManager(layoutManager);

        recyclerAdapter = new RecyclerAdapter(images);
        recyclerView.setAdapter(recyclerAdapter);



    }
}

File: Razvoj mobilnih aplikacija labosi/BazaIRecyclerView/app/src/main/java/com/filip/bazairecyclerview/MainActivity.kt
package com.filip.bazairecyclerview

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v7.widget.LinearLayoutManager
import android.support.v7.widget.RecyclerView
import android.util.Log
import android.widget.LinearLayout
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    private val bookAdapter by lazy { BookAdapter() }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        tapMe.setOnClickListener { addBook() }
        display()

        setUpUi()
    }

    private fun addBook() {
        val input = editText.text.toString()
        val book = Book("$input", "$input")
        BookDatabase.bookDao.insert(book)
        Log.d("dsds", book.toString())
        display()

    }

    private fun display() {
        bookAdapter.setData(BookDatabase.bookDao.getAll())
    }

    private fun setUpUi() {
        recyclerView.layoutManager = LinearLayoutManager(this)
        recyclerView.adapter = bookAdapter
    }
}
